cmake/modules/tcmalloc.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 Husky Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+### TCMalloc ###
+
+# TODO(legend): check zeromq version?
+find_library(TCMALLOC_LIBRARY NAMES tcmalloc)
```

Comment:
You can add `find_path(TCMALLOC_INCLUDE_DIR NAMES gperftools/malloc_extension.h)` in front of `find_library` in your own `cmake/dep.cmake` for test.